lib/cap_authors_poller.rb

```diff
@@ -126,6 +126,11 @@ def queue_author_for_harvest(author, skip_message)
   end
 
   def update_existing_contributions(author, incoming_authorships)
+    # inject author identity into the authorship data
+    incoming_authorships = incoming_authorships.dup.map do |contrib|
```

Comment:
does this mean we would store the injected cap_profile_id in this data structure, or is this purely to pass validation?